packages/devextreme/js/__internal/ui/chat/chat_message_bubble.ts

```diff
@@ -18,22 +18,28 @@ class MessageBubble extends Widget<MessageBubbleOptions> {
   }
 
   _initMarkup(): void {
-    const $bubble = $(this.element()).addClass(CHAT_MESSAGE_BUBBLE_CLASS);
+    $(this.element())
+      .addClass(CHAT_MESSAGE_BUBBLE_CLASS);
 
-    const { text } = this.option();
+    super._initMarkup();
 
-    if (text) {
-      $bubble.text(text);
-    }
+    this._updateText();
+  }
 
-    super._initMarkup();
+  _updateText(): void {
+    const { text } = this.option();
```

Comment:
[ts]
We can easily get rid of this ts error
```suggestion
    const { text = '' } = this.option();
```

Reply:
don't have any strict objection to it.